eosio/Dockerfile

```diff
@@ -4,18 +4,19 @@ RUN echo "INSTALLING EOSIO AND CDT"
 RUN apt-get update && apt-get install -y wget sudo curl
 RUN wget https://github.com/EOSIO/eosio.cdt/releases/download/v1.6.1/eosio.cdt_1.6.1-1_amd64.deb
 RUN apt-get update && sudo apt install -y ./eosio.cdt_1.6.1-1_amd64.deb
-RUN wget https://github.com/EOSIO/eos/releases/download/v1.7.3/eosio_1.7.3-1-ubuntu-18.04_amd64.deb
-RUN apt-get update && sudo apt install -y ./eosio_1.7.3-1-ubuntu-18.04_amd64.deb
-
+#RUN wget https://github.com/EOSIO/eos/releases/download/v1.8.3/eosio_1.8.3-1-ubuntu-18.04_amd64.deb
+#RUN apt-get update && sudo apt install -y ./eosio_1.8.3-1-ubuntu-18.04_amd64.deb
+COPY ./eosio_1.9.0-develop_amd64.deb ./
+RUN apt-get update && sudo apt install -y ./eosio_1.9.0-develop_amd64.deb
 RUN echo "INSTALLING CONTRACTS"
 RUN mkdir -p "/opt/eosio/bin/contracts"
 
 RUN echo "INSTALLING EOSIO.CONTRACTS"
-RUN wget https://github.com/EOSIO/eosio.contracts/archive/v1.6.0.tar.gz
+RUN wget https://github.com/EOSIO/eosio.contracts/archive/v1.7.0.tar.gz
 RUN mkdir -p /eosio.contracts
-RUN tar xvzf ./v1.6.0.tar.gz -C /eosio.contracts
-RUN mv /eosio.contracts/eosio.contracts-1.6.0 /opt/eosio/bin/contracts
-RUN mv /opt/eosio/bin/contracts/eosio.contracts-1.6.0 /opt/eosio/bin/contracts/eosio.contracts
+RUN tar xvzf ./v1.7.0.tar.gz -C /eosio.contracts
+RUN mv /eosio.contracts/eosio.contracts-1.7.0 /opt/eosio/bin/contracts
+RUN mv /opt/eosio/bin/contracts/eosio.contracts-1.7.0 /opt/eosio/bin/contracts/eosio.contracts
```

Comment:
eosio 1.9 and contracts 1.7.  Just want to confirm those (different) numbers are correct

Reply:
Yes, though eosio `1.9.0` will never be released.  Instead that is the development version for what is to be `2.0.0` .   Eventually, this will revert to downloading the `>=2.0-rc1` packages for eosio.   It may pull a later version of contracts but the current stable release (`1.7.0`) is sufficient to activate the needed consensus upgrade feature.